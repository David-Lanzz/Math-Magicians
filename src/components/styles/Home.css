.Home {
  display: flex;
  flex-direction: column;
  text-align: center;
  gap: 1rem;
}

.Home h3 {
  font-size: 1.5rem;
}

.Home p {
  margin-bottom: 1rem;
}

@media screen and (min-width: 768px) {
  .Home {
    padding: 4% 10%;
    text-align: start;
  }
}
