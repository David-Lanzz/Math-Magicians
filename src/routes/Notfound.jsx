const Notfound = () => (
  <>
    <h2>The page you&apos;re looking for does not exist...</h2>
  </>
);

export default Notfound;
