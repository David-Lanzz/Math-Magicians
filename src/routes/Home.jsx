import Homecontent from '../components/homecontent';
import styles from '../components/styles/Home.module.css';

const Home = () => (
  <div className={styles.Home}>
    <h3>Welcome to our page!</h3>
    <Homecontent />
  </div>
);

export default Home;
