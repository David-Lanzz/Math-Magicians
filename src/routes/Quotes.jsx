import Getquotes from '../components/quotes';
import styles from '../components/styles/Quotes.module.css';

const Quotes = () => (
  <div className={styles.get}>
    <Getquotes />
  </div>
);
export default Quotes;
