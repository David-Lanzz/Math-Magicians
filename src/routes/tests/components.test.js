import renderer from 'react-test-renderer';
import Home from '../Home';
import Quotes from '../Quotes';
import Notfound from '../Notfound';
import CalculatorPage from '../Calculator';

describe('All components tests', () => {
  test('correct snapshot for ', () => {
    const tree = renderer.create(<CalculatorPage />);
    const test = tree.toJSON();
    expect(test).toMatchSnapshot();
  });
  test('correct snapshot for ', () => {
    const tree = renderer.create(<Notfound />);
    const test = tree.toJSON();
    expect(test).toMatchSnapshot();
  });
  test('correct snapshot for ', () => {
    const tree = renderer.create(<Home />);
    const test = tree.toJSON();
    expect(test).toMatchSnapshot();
  });
  test('correct snapshot for ', () => {
    const tree = renderer.create(<Quotes />);
    const test = tree.toJSON();
    expect(test).toMatchSnapshot();
  });
});
