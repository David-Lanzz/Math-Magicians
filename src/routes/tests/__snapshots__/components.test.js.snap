// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`All components tests correct snapshot for  1`] = `
<div
  className="body1"
>
  <h3>
    Let's do some Math!
  </h3>
  <div
    className="container"
  >
    <input
      placeholder="  "
      readOnly={true}
      type="text"
    />
    <div
      className="keys"
    >
      <button
        id={0}
        name="AC"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        id={1}
        name="+/-"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        id={2}
        name="%"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        id={3}
        name="รท"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
      <button
        id={4}
        name={7}
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        id={5}
        name={8}
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        id={6}
        name={9}
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        id={7}
        name="x"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        id={8}
        name={4}
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        id={9}
        name={5}
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        id={10}
        name={6}
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        id={11}
        name="-"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        id={12}
        name={1}
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        id={13}
        name={2}
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        id={14}
        name={3}
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        id={15}
        name="+"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        id={16}
        name={0}
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        id={17}
        name="."
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        id={18}
        name="="
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;

exports[`All components tests correct snapshot for  2`] = `
<h2>
  The page you're looking for does not exist...
</h2>
`;

exports[`All components tests correct snapshot for  3`] = `
<div
  className="Home"
>
  <h3>
    Welcome to our page!
  </h3>
  <div>
    <p>
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quos architecto cum voluptas esse consequatur optio maxime molestiae, nihil incidunt temporibus iste asperiores praesentium sapiente. Ut quisquam nulla dignissimos officia temporibus.
    </p>
    <p>
      Lorem ipsum, dolor sit amet consectetur adipisicing elit. Aperiam dicta inventore neque, officiis culpa error eveniet minus, veniam reprehenderit rerum quasi commodi iusto animi ab quis eaque delectus illum nam!
    </p>
  </div>
</div>
`;

exports[`All components tests correct snapshot for  4`] = `
<div
  className="get"
>
  <div>
    <p />
  </div>
</div>
`;
